src/global.d.ts

```diff
@@ -0,0 +1,9 @@
+// evanw/esbuild#2388
+
+declare global {
+  namespace WebAssembly {
+    interface Module {}
+  }
+}
```

Comment:
Oops, forgot one thing: we need to figure out a way to include declaration in skuba's bundle and for TypeScript to pick it up in consuming repos, else you'll get compiler errors when type checking esbuild itself unless you have `DOM` types or `skipLibCheck: true`.

Reply:
I think 1494857 will do it, but this upcoming release probably necessitates a beta or two to ensure everything comes together correctly.